README.md

```diff
@@ -50,6 +50,19 @@ Once your app is up and running with these variables in place, you should be abl
 
 This repository is essentially a minimal web application that specifies [Ghost as a dependency](https://github.com/TryGhost/Ghost/wiki/Using-Ghost-as-an-NPM-module), and makes a deploy button available.
 
+
+## Updating
+
+After deploying your own ghost blog, you can update it by running the following commands:
+<pre><code>heroku clone --app YOURAPPNAME && cd YOURAPPNAME
+git remote add origin https://github.com/cobyism/ghost-on-heroku
+git pull origin master # may trigger a few merge conflicts, depending on how long since last update
+git push heroku master
+</code></pre>
```

Comment:
Instead of the `<pre><code></code></pre>` wrapper, I'd recommend using Markdown's code blocks (three backticks) like the rest of the README, e.g.:

    ```
    heroku clone --app YOURAPPNAME && cd YOURAPPNAME
    git remote add origin https://github.com/cobyism/ghost-on-heroku
    git pull origin master # may trigger a few merge conflicts, depending on how long since last update
    git push heroku master
    ```